TwilioVerifyDemo/TwilioVerifyDemo/FactorDetail/View/FactorDetailViewController.swift

```diff
@@ -19,7 +19,7 @@
 
 import UIKit
 
-protocol FactorDetailView: class {
+protocol FactorDetailView: AnyObject {
```

Comment:
Why is the reason of this change?

Reply:
Using `class` for protocols is deprecated, I couldn't find the official source but here is the warning: 
![image](https://user-images.githubusercontent.com/14830877/133322026-e7c2a44c-2811-4133-a970-29a88bcd769b.png)